doc/release-notes/8578-support-extended-iso-639-languages.md

```diff
@@ -0,0 +1,12 @@
+The Controlled Vocabulary Values list for the metadata field Language in the Citation block has been extended.
+Roughly 300 ISO 639 languages added.
```

Comment:
Why only 300? Where did that subset come from? Are these 639-2 codes (guessing from zho which is in 639-2 and as far as I can see not in 639-3)?

Reply:
The list came from the Library of Congress. Yes, they are 639-2 codes. Once I merged all the codes and removed the duplicates the list increased by ~300.

Reply:
The PR says that it "closes #8578". That issues is specifically for figuring out how to support ISO **639-3** list (*thousands* of languages). All the recent discussion in that issue was centered around that - whether it was feasible to just add that many to the CVV, whether the pulldown menu was still going to be usable in the UI and/or whether we wanted to consider using an external CV instead, etc. 
I can understand it if we decide to handle this via incremental improvements, and first extend the list to the full 639-2, before proceeding with the 639-3. But I'm not seeing any discussion of this approach neither in #8578 nor in this PR. And, at the very least, I don't think this PR should be closing that issue. 

Please note that there is real interest in the full ISO 639-3 support, see, for example, #10481, where a user is specifically requesting it.

Reply:
I can pull this feature back and load the full 639-3 languages. The file seems like a lot but if I remember correctly it's one line per language and we combine the language codes so the final list would be shorter. Let me take another look at this.
Moving back to in progress